frontend/src/components/Cell.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Badge from "./Badge";
+
+export default (props: {
+  color: string;
+  label: string;
+  bgcolor: string;
+  index: number;
+  onClick: (date: Date, index: number) => void;
+  dateObj: Date;
+  hasNote: boolean;
+}) => {
+  const col = `text-${props.color} bg-${props.bgcolor}`;
+
+  return (
+    <div
+      className={`w-[50px] h-[40px] flex flex-col justify-center items-center border-none select-none cursor-pointer ${col}`}
```

Comment:
Instead of adding the dynamic string to the classnames manually, You should use `classname` module.

Reply:
In dynamic strings, we can only add string-formatted class names, but in case of classname module, we can use objects, arrays and strings. All of them.
So it's very helpful when we combine several class names from different sources.